qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteCodeActions.java

```diff
@@ -397,4 +441,128 @@ private static void doCodeActionsForUndefinedSectionTag(Template template, Diagn
 			LOGGER.log(Level.SEVERE, "Creation of undefined user tag code action failed", e);
 		}
 	}
+
+	private void doCodeActionsForSimilarValues(Template template, Diagnostic diagnostic, List<CodeAction> codeActions,
+			String varName) throws BadLocationException {
+		Range diagnosticRange = diagnostic.getRange();
+		int offset = template.offsetAt(diagnosticRange.getEnd());
+
+		Node nodeBefore = template.findNodeBefore(offset);
+		if (nodeBefore == null) {
+			return;
+		}
+
+		Node node = QutePositionUtility.findBestNode(offset, nodeBefore);
+
+		if (node.getKind() == NodeKind.ExpressionPart) {
+			Range rangeValue = new Range(
+					new Position(diagnosticRange.getStart().getLine(), diagnosticRange.getStart().getCharacter()),
+					new Position(diagnosticRange.getEnd().getLine(), diagnosticRange.getEnd().getCharacter()));
+
+			Collection<String> availableValues = new TreeSet<>();
+
+			switch (((Part) node).getPartKind()) {
+			case Object:
+				collectAvailableValuesForObjectPart(node, template, availableValues);
+			case Namespace:
+				collectAvailableValuesForNamespacePart(node, template, availableValues);
+				break;
+			case Method:
+			case Property:
+				String signature = null;

```

Comment:
Please remove this code which support similar value for method/property which doesn't work. We will see that in an another PR.